src/Catalog/sparql/ConstructRegistrationPageContentGraph.rq

```diff
@@ -57,8 +59,6 @@ WHERE
 	OPTIONAL { ?catalogEntry nuget:description ?optionalDescription . }
 	OPTIONAL { ?catalogEntry nuget:title ?optionalTitle . }
 	OPTIONAL { ?catalogEntry nuget:summary ?optionalSummary . }
-	OPTIONAL { ?catalogEntry nuget:iconUrl ?optionalIconUrl . }
-	OPTIONAL { ?catalogEntry nuget:licenseUrl ?licenseUrl . }
```

Comment:
Is deleting the license intended?

Reply:
Previously, we will set an empty licenseUrl string in the registration. Client will decide whether to show the licenseUrl base on it's empty or not. If we don't inject the licenseUrl if the package has no license info, client should have the logic to handle this case rather than expect to check whether it's empty or not.

Reply:
Yes, it was intended. Removed lines did nothing, the value is always provided from the code.